skimage/segmentation/__init__.py

```diff
@@ -5,6 +5,7 @@
 from ._quickshift import quickshift
 from .boundaries import find_boundaries, mark_boundaries
 from ._clear_border import clear_border
+from .chan_vese import chan_vese
```

Comment:
Causes an error for Appveyor.
`from .chanvese import chan_vese` should be the correct version.
Although I'd suggest to be uniform and use either `chan_vese` or `chanvese` everythere across the PR.